addons/hr/models/res_users.py

```diff
@@ -160,6 +160,15 @@ def fields_view_get(self, view_id=None, view_type='form', toolbar=False, submenu
                 result['toolbar']['action'] = [act for act in result['toolbar']['action'] if act['id'] != change_password_action.id]
         return result
 
+    def get_employee_values_sync(self, vals):
+        """This method keep the original behavior inside 'write' method, but allow us to select either a
```

Comment:
This is more of a commit description than a docstring. If read after your change is applied, it doesn't make much of a sense.

Maybe something like

```"""Get values to synk with related employee when the user is changed"""